framework/include/mesh/MooseMesh.h

```diff
@@ -1842,6 +1851,14 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   /// faces
   bool _has_lower_d;
 
+  /// Indicates whether any disconnected boundary pairs are incomplete,
+  /// i.e., only one side of an expected (blockA, blockB) / (blockB, blockA) pair exists.
```

Comment:
In this line we're adding block language, inconsistent with the previous line. Perhaps the best language to use is "sidesets" to indicate the fact that we are pairing boundaries but that these boundaries are fundamentally linked with their element faces. Sidesets certainly fit with cohesive zone modeling using interface kernels

Reply:
Oh wait I see it below. I think you probably meant to delete this doxygen block and the `_has_incomplete_disjoint_block_sidesets` data member?

Reply:
@lindsayad correct.

Reply:
Removed this flag.